src/ring_theory/polynomial/basic.lean

```diff
@@ -1062,10 +1125,11 @@ def quotient_equiv_quotient_mv_polynomial (I : ideal R) :
 
 end mv_polynomial
 
-namespace polynomial
+section unique_factorization_domain
+variables {D : Type u} [comm_ring D] [is_domain D] [unique_factorization_monoid D] (σ)
 open unique_factorization_monoid
 
-variables {D : Type u} [comm_ring D] [is_domain D] [unique_factorization_monoid D]
+namespace polynomial
 
 @[priority 100]
```

Comment:
Not sure why we need this, but I copied it blindly to the mv_polynomial case below.